static_src/test/unit/util/cf_api.spec.js

```diff
@@ -470,6 +470,29 @@ describe('cfApi', function() {
     });
   });
 
+  describe('fetchAppAll()', function () {
+    it('should call fetchApp() and fetchAppStats()', function () {
+      const guid = 'shouldCallTwoFnsGuid';
+      const fetchAppSpy = sandbox.stub(cfApi, 'fetchApp');
+      const fetchAppStatsSpy = sandbox.stub(cfApi, 'fetchAppStats');
+
+      cfApi.fetchAppAll(guid);
+
+      expect(fetchAppSpy).toHaveBeenCalledOnce();
+      expect(fetchAppStatsSpy).toHaveBeenCalledOnce();
+    });
+
+    it('should call the receivedAppAll app action', function (done) {
+      const guid = 'shouldCallActionCreatorGuid';
+      const actionCreatorSpy = sandbox.spy(appActions, 'receivedAppAll');
+
+      cfApi.fetchAppAll(guid).then(() => {
+        expect(actionCreatorSpy).toHaveBeenCalled();
```

Comment:
Hmm, I can't think of anything. When I have problems with things like this, I'll put a `debugger` statement in the action or maybe the spy and then run in in the browser with `watch-test`. Then I'll try and get a good stack trace of whos calling(it doesn't always give me a good stack track). We can check it out tomorrow.